amstrax/auto_processing/auto_processing.py

```diff
@@ -58,7 +58,7 @@ def parse_args():
 
         for run_doc in run_docs_to_do[:max_jobs]:
             run_name = f'{int(run_doc["number"]):06}'
-            submit_stbc.submit_job(run_name, target=target, script='process_run', context=context, detector=detector)
+            submit_stbc.submit_job(run_name, target=target, context=context, detector=detector)
```

Comment:
I am not sure about this. The process_run.py script is a wrapper around the amstraxer script for a single run processing and it is helpful to monitor the update of the runs processing in the database. I would prefer to leave it as it was before, unless I miss something?

Reply:
I agree, I was too fast :)